.github/workflows/test-gh.yml

```diff
@@ -29,40 +30,53 @@ jobs:
         path: src/github.com/${{ github.repository }}
         fetch-depth: 0
     - name: Run Tests
+      env:
+        DOCKERHUB_USERNAME: k8slt
+        DOCKERHUB_ACCESS_TOKEN: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
       run: |
         set -e -x
 
         mkdir -p ~/.docker/machine/cache
-        curl -Lo ~/.docker/machine/cache/boot2docker.iso https://github.com/boot2docker/boot2docker/releases/download/v19.03.12/boot2docker.iso
+        wget -O- -nv https://github.com/boot2docker/boot2docker/releases/download/v19.03.12/boot2docker.iso > ~/.docker/machine/cache/boot2docker.iso
+
         brew install docker docker-machine
 
         # Install ytt, kapp for build and tests
         mkdir -p /tmp/bin
         export PATH=/tmp/bin:$PATH
 
-        wget -O- https://k14s.io/install.sh | K14SIO_INSTALL_BIN_DIR=/tmp/bin bash
+        wget -O- -nv https://k14s.io/install.sh | K14SIO_INSTALL_BIN_DIR=/tmp/bin bash
 
-        wget -O- https://github.com/kubernetes/minikube/releases/download/v1.10.0/minikube-darwin-amd64 > /tmp/bin/minikube
+        wget -O- -nv https://github.com/kubernetes/minikube/releases/latest/download/minikube-darwin-amd64 > /tmp/bin/minikube
         chmod +x /tmp/bin/minikube
-        minikube start --driver=virtualbox --insecure-registry=192.168.99.100/16
+        minikube start --driver=virtualbox --insecure-registry=192.168.0.0/16
         eval $(minikube docker-env --shell=bash)
 
+        # Authenticate to DockerHub to avoid test failures due to rate limiting
+        # ... for builders using Docker
+        echo "${DOCKERHUB_ACCESS_TOKEN}" | docker login --username "${DOCKERHUB_USERNAME}" --password-stdin
```

Comment:
With the above bash option`set -x` I think bash will print out the eval'd `DOCKERHUB_ACCESS_TOKEN` variable. 

for e.g.
```
bash-5.0$ FOO=test
bash-5.0$ set -x
bash-5.0$ echo $FOO | grep bar
+ echo test
+ grep bar
bash-5.0$
```

But I'm pretty sure github actions will redact this. But!... If you wanted to be safer than sorry you could wrap this around `set +x / set -x`, or potentially drop the `set -x` on top?

Reply:
Yeah, it _is_ redacted, but not interested in relying on that. I'll wrap the call as it's particularly helpful in troubleshooting to have `set -x`... and we've done _other_ work to significantly reduce the total log size. 👍🏻

Reply:
that makes sense! i'll resolve this convo :-)